cli/medperf/utils.py

```diff
@@ -228,6 +228,12 @@ def combine_proc_sp_text(proc: spawn) -> str:
     static_text = ui.text
     proc_out = ""
     break_ = False
+    log_pattern = re.compile(
+        r"^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} \S+ \S+\[(\d+)\] (DEBUG|INFO) (.*)$"
```

Comment:
we should only hide DEBUG logs. INFO should be shown

Reply:
Ah, I see, you changed the default mlcube log level
Fixed 👌